website/content/api-docs/system/lease-count-quotas.mdx

```diff
@@ -17,6 +17,12 @@ A lease count quota must include a `max_leases` value with an optional `path`
 that can either be a namespace or mount, and can optionally include a path suffix following
 the mount to restrict more specific API paths.
 
+Upon creating a lease count quota, it will be populated with the current count of leases from this path. If there are
+more leases generated than the specified `max_leases`, this will cause the lease count to go over the specified
```

Comment:
```suggestion
more leases present than the specified `max_leases`, this will cause the lease count to go over the specified
```

I prefer this wording because "generated" has a bit of ambiguity as to when the generation happens.